metadata/glyphnames.json

```diff
@@ -3886,7 +4070,7 @@
 	},
 	"figbass7Raised2": {
 		"codepoint": "U+EA5F",
-		"description": "Figured bass 7 raised by a half-step 2"
+		"description": "Figured bass 7 lowered by a half-step"
```

Comment:
Hi, just wanted to double check that this change (from "raised" to "lowered") was done on purpose since I could not find this mentioned in any issue.

Reply:
That changed text doesn't seem to make much sense for a symbol called `figbass7Raised2`, or does it?

Reply:
Unfortunately we cannot change canonical glyph names, even if we later discover that we made an error when classifying the symbol. There are handful of such errors in the specification, and perhaps one day we could come up with a way of making it possible to fix these without introducing major problems for existing applications.

Reply:
so the canonical name is wrong? But isn't this the private use area which does not need the full Unicode rigor? If not, maybe the description should mention the mismatch between name and description?

Reply:
Yes, the canonical name is wrong. There are a few such examples (e.g. see the new issue #167) but without agreement from the community about fixing these glyph names, we can't do so.